### PR TITLE
feat(int-805): Generate single file for pull-components + Login with token using a flag --token

### PR DESCRIPTION
**Pull request by ademarCardoso**
## Pull request type

Jira Link: [INT-805](https://storyblok.atlassian.net/browse/INT-805)

- [x] Feature

## How to test this PR

In the video bellow you can see i'm using the two ways to download the components, with the `--single-file` flag and without

https://user-images.githubusercontent.com/40925579/221249097-d15dcfc1-097d-45ff-96ea-6bd3e0902396.mov

In the video bellow you can see the new login strategy adding the new flag --token and --region to enable the login with one single command

https://user-images.githubusercontent.com/40925579/221249172-80d63e0a-c036-40b9-830b-8e17091e02d9.mov

## What is the new behavior?
- Now it's possible to generate single files for the pull-components command adding a flag `--sf, --single-file`
- And it's possible to pass one single file for the push-components like: 
  - `# storyblok push-components ./page-1234.json --space SPACE_ID` or multiple files `# storyblok push-components ./page-1234.json,./feature-1234.json --space SPACE_ID`
- Now it's possible to login in the CLI using token like `# storyblok login --token 1234 --region eu` without respond the questions
- I update the Readme file with the new flags and behaviors

## Other information

[INT-805]: https://storyblok.atlassian.net/browse/INT-805?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by markus-gx** [comment]
Wouldn't "--separate-files" be more accurate than "single-file" ? 🤔 @ademarCardoso
Was quite confused when I read the change :o

**Comment by ademarCardoso** [comment]
> Wouldn't "--separate-files" be more accurate than "single-file" ? 🤔 @ademarCardoso Was quite confused when I read the change :o

Yes my man, thank you for the tip, you are correct, i fixed the name 😄

File: README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ $ storyblok login
 
 * `region`: your user's region (default: `eu`). You can use `us`, `cn` or `eu`. This region will be used for the other cli's commands.
 
+**You also could login in the CLI adding the token directly in the login command without waiting and adding the information in the questions, like:**
```
Comment (thiagosaife) [inline README.md]: @ademarCardoso This sentence is not very clear to newcomers.

**Review by thiagosaife — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ademarCardoso) [reply README.md]: Fixed 😄

**Pull request merged**